Capitalize “M” in Mate (as in MeshMates )

**Current Behavior**
The references to the word **MeshMate**  have the M in  Mate in small letter.


**Desired Situation**

All references to MeshMates on this site should have the “M” in Mate capitalized.
https://layer5.io/community/meshmates 
![Screenshot 2021-06-03 at 15 31 15](https://user-images.githubusercontent.com/62384659/120663472-f5499e80-c481-11eb-94af-e677021406bd.png)



---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

Hi, please help!
I am trying to run the code locally and i gave the command "make site" and got the following error. 
npm ERR! node v14.17.0
npm ERR! npm  v3.10.5
npm ERR! code ELIFECYCLE
npm ERR! Layer5@1.0.0 start: `npm run develop`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the Layer5@1.0.0 start script 'npm run develop'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the Layer5 package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run develop
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs Layer5
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls Layer5

> Hi, please help!
> I am trying to run the code locally and i gave the command "make site" and got the following error.
> npm ERR! node v14.17.0
> npm ERR! npm v3.10.5
> npm ERR! code ELIFECYCLE
> npm ERR! Layer5@1.0.0 start: `npm run develop`
> npm ERR! Exit status 1
> npm ERR!
> npm ERR! Failed at the Layer5@1.0.0 start script 'npm run develop'.
> npm ERR! Make sure you have the latest version of node.js and npm installed.
> npm ERR! If you do, this is most likely a problem with the Layer5 package,
> npm ERR! not with npm itself.
> npm ERR! Tell the author that this fails on your system:
> npm ERR! npm run develop
> npm ERR! You can get information on how to open an issue for this project with:
> npm ERR! npm bugs Layer5
> npm ERR! Or if that isn't available, you can get their info via:
> npm ERR! npm owner ls Layer5

Hey,are you stuck with this issue,maybe i can help?

--- a/src/sections/Community/Meshmates/index.js
+++ b/src/sections/Community/Meshmates/index.js
@@ -49,7 +49,7 @@ const Meshmates=() => {
         <Row className="hero">
           <Col sm={12} lg={6}>
             <h5>A warm and welcoming collection of open sourcers</h5>
-            <h1>Layer5 Meshmates</h1>
+            <h1>Layer5 MeshMates</h1>
             <p>
                   Layer5 MeshMates are committed to helping community members be successful contributors.
                   MeshMates aid in identifying areas of projects to engage within, working groups to join,
@@ -62,7 +62,7 @@ const Meshmates=() => {
           </Col>
         </Row>
         <div className="meshmates-grid">
-          <h1> Our Community Meshmates </h1>
+          <h1> Our Community MeshMates </h1>
           <p> MeshMate is a distinction that Layer5 awards select members of the community that who innately align with the Layer5 culture of helping others, paying it forward, and have a commitment to sharing their knowledge of Layer5 projects with the community. MeshMates are Layer5 ambassadors (not employees) and their commitment to helping others and sharing their expertise has a huge impact on the Layer5 community – don’t hesitate to reach out to them! </p>
           <Row>
             {data.allMdx.nodes.map(({ id, frontmatter, fields }) => (
@@ -81,7 +81,7 @@ const Meshmates=() => {
         <div className="meshmate-meet">
           <Row Vcenter>
             <Col sm={12} lg={6}>
-              <h2> Meeting Your Meshmate </h2>
+              <h2> Meeting Your MeshMate </h2>
               <p> Slack’s video chat or Google Hangouts are both available for your use as tools for getting to know one another. While getting acquainted and onboarding into the community, we suggest the following goals: </p>
               <table>
                 <tr>

